crates/sui-deepbook-indexer/src/migrations/00000000000000_diesel_initial_setup/up.sql

```diff
@@ -178,4 +180,22 @@ CREATE TABLE IF NOT EXISTS sui_error_transactions
     failure_status              TEXT         NOT NULL,
     package                     TEXT         NOT NULL,
     cmd_idx                     BIGINT
-);
\ No newline at end of file
+);
+
+CREATE TABLE IF NOT EXISTS pools
+(
+    id                          SERIAL       PRIMARY KEY,
```

Comment:
is there a unique identifier we can use for each pool (i'm looking ay `pool_id`)? By using a meaningful unique identifier as primary key we can save one unique key restriction.